aips/higher-order-functions.md

```diff
@@ -0,0 +1,132 @@
+---
+aip: TBD
+title: Higher-Order Inline Functions for Collections
+author: wrwg
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/33
+Status: Draft
+last-call-end-date: TBD
+type: Standard (Framework)
+created: 2023/1/9
+updated: 2023/1/9
+---
+
+## Summary
+
+Recently, the concept of *inline functions* has been added to Aptos Move.
+Those functions are expanded at compile time and do not have an equivalent
+in the Move bytecode. This allows them to implement a feature which is
```

Comment:
"this" and "them" don't read clearly
might recommend something like
"This offers the ability to implement"
or
"Inline functions allow us to implement a feature currently unavailable for ..."